pkg/schedule/schedulers/diagnostic_recorder.go

```diff
@@ -131,11 +132,11 @@ func (d *DiagnosticRecorder) GetLastResult() *DiagnosticResult {
 			}
 		} else if firstStatus == Pending {
 			// This is used to handle pending status because of reach limit in `IsScheduleAllowed`
-			resStr = fmt.Sprintf("%s reach limit", d.schedulerName)
+			resStr = fmt.Sprintf("%v reach limit", d.schedulerType)
 		}
 	}
 	return &DiagnosticResult{
-		Name:      d.schedulerName,
+		Name:      string(d.schedulerType),
```

Comment:
What is the difference between `string(d.schedulerType)` and `d.schedulerType.String()`?

Reply:
no difference

Reply:
Then shall we keep consistent with other places?

Reply:
updated